package.json

```diff
@@ -42,20 +42,29 @@
   "dependencies": {
     "bluebird": "^3.1.4",
     "body-parser": "^1.14.2",
+    "compose-middleware": "^2.2.0",
     "compression": "^1.6.1",
     "cookie-parser": "^1.4.1",
     "cors": "^2.7.1",
-    "dotenv": "^2.0.0",
     "cross-env": "^2.0.1",
+    "crypto": "0.0.3",
+    "dotenv": "^2.0.0",
     "express": "~4.13.1",
+    "express-jwt": "^5.1.0",
```

Comment:
What do you think about use "node-jsonwebtoken" instead "express-jwt" ? "express-jwt" package uses as dependency "node-jsonwebtoken" so do we really need additional wrapper ?